src/border_agent/border_agent.cpp

```diff
@@ -130,7 +130,7 @@ struct StateBitmap
         , mBbrIsActive(0)
         , mBbrIsPrimary(0)
         , mThreadRole(kThreadRoleDisabledOrDetached)
-        , mEpskcSupported(OTBR_ENABLE_EPSKC)
+        , mEpskcSupported(0)
```

Comment:
How about enable by default if OPENTHREAD_CONFIG_THREAD_VERSION is OT_THREAD_VERSION_1_4?

Reply:
To achieve this, you may consider initialise this variable according to the value returned from [otThreadGetVersion](https://github.com/openthread/openthread/blob/aba7aede1476339abe3032a250349d67a4ec1605/include/openthread/thread.h#L252)

Reply:
Accept，add in the initiation, let me know there is more concern. thanks~